pkg/sip/server.go

```diff
@@ -218,7 +218,19 @@ func (s *Server) Start(agent *sipgo.UserAgent, unhandled sipgo.RequestHandler) e
 	s.sipUnhandled = unhandled
 
 	// Ignore ACKs
-	s.sipSrv.OnAck(func(req *sip.Request, tx sip.ServerTransaction) {})
+	s.sipSrv.OnAck(func(req *sip.Request, tx sip.ServerTransaction) {
+	        fromTag, err := getTagValue(req)
+	        if err != nil {
+	            sipErrorResponse(tx, req)
+	            return
+	        }
+	
+	        if c, found := s.activeCalls[fromTag]; found {
```

Comment:
```suggestion
	        s.cmu.RLock()
	        c, found := s.activeCalls[fromTag]
	        s.cmu.RUnlock()
	        if found {
```
Must hold mutex when accessing `activeCalls`.